pyVmomi/SoapAdapter.py

```diff
@@ -1208,6 +1208,8 @@ def __init__(self, host='localhost', port=443, ns=None, path='/sdk',
       if cacertsFile:
          self.schemeArgs['ca_certs'] = cacertsFile
          self.schemeArgs['cert_reqs'] = ssl.CERT_REQUIRED
+      if self.schemeArgs == {}:
+         self.schemeArgs['context'] = ssl._create_unverified_context()
```

Comment:
My only comment here is to figure out how to incorporate the other work on switches to allow the class to toggle between this unverified default behavior and the stronger verified behavior that the security folks want to see into this patch.